### PR TITLE
 fix: wrong usd price of dot coin in purchase cart

### LINKED ISSUE #6943: Not calculating usd total amount in confirmation modal
image
image

### LINKED ISSUE #6961: Wrong Price in items explorer
### What happened?

DOT price and usd price are not matched in shopping cart and in confirm purchase modal
USD price is 100X lower then it should be

### Please reproduce in steps

1.go here:
/ahp/gallery/0-18

add it to shopping cart
USD price should be 100X more (in shopping cart)

2. go to any item in ahp explore
/ahp/explore/items?listed=true

click ' buy now"
USD price in the confirm purchase modal should be 100X more

### Expected Behavior

matching price

### What browsers are you seeing the problem on?

_No response_

### At which address did you encounter bug?

_No response_

### Are you logged in?

None

### Which wallet you are using?

_No response_

### At which chain did you encounter bug?

_No response_

### Screenshots

image

image

image

### Relevant log output

_No response_

### Payment link for reward

https://kodadot.xyz/ksm/transfer/?target=%3CMy_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address%3E

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by xiiiAtCn**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #6961 
- [x] Closes #6943 

#### Did your issue had any of the "$" label on it?
My DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=12jXHf7yYF4wTRjDPbwFB24V1nVRPRkuU2S8Ke5ZWTfvyBGW&usdamount=8.82&donation=true)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

shot

### <samp>🤖 Generated by Copilot at cf06fe5</samp>

Improved token decimal handling in `shoppingCart` component. Used `getChainProperties` function to convert token prices and royalties to USD in `components/common/shoppingCart/utils.ts`.

### <samp>🤖 Generated by Copilot at cf06fe5</samp>

> _Sing, O Muse, of the skillful coder who refined_
> _the shopping cart component with a clever mind._
> _He imported a function to get the chain properties_
> _of tokens, and converted their prices and royalties._

### CODE REVIEW

**Review by Jarsen136 — APPROVED** [review]

File: components/common/shoppingCart/utils.ts [hunk 1]
```diff
@@ -14,20 +15,26 @@ export const prefixToToken = {
   dot: 'DOT',
 }
 
+const getTokenDecimal = (item: ShoppingCartItem) => {
+  const token = prefixToToken[item.urlPrefix]
+  const tokenDecimal = chainPropListOf(token.toLowerCase()).tokenDecimals
+  return tokenDecimal
```
Comment (daiagi) [inline components/common/shoppingCart/utils.ts:21]:
```suggestion
return chainPropListOf(token.toLowerCase()).tokenDecimals
```

**Review by daiagi — APPROVED** [review]
wfm

**Review by preschian — APPROVED** [review]

**Review by stephenjason89 — APPROVED** [review]
looks good

**Comment by yangwao** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 4.44 USD/DOT ~ 6.757 $DOT
🧗 [12jXHf7yYF4wTRjDPbwFB24V1nVRPRkuU2S8Ke5ZWTfvyBGW](https://polkadot.subscan.io/account/12jXHf7yYF4wTRjDPbwFB24V1nVRPRkuU2S8Ke5ZWTfvyBGW)
🔗 [0xa7098f08c02295f7ad0a70f30af59a7437d91da886f432023bbffe64f4364135](https://polkadot.subscan.io/extrinsic/0xa7098f08c02295f7ad0a70f30af59a7437d91da886f432023bbffe64f4364135)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**